test-harness/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "ata-mollusk-harness"
+version = "1.0.0"
+edition = "2021"
+license = "Apache-2.0"
+
+[lib]
+path = "src/lib.rs"
+
+[dependencies]
+mollusk-svm = { version = "0.6.0" }
+solana-account = "3.0"
+solana-instruction = "3.0"
+solana-keypair = "3.0"
+solana-program-error = "3.0"
+solana-program-option = "3.0"
+solana-program-pack = "3.0"
+solana-pubkey = "3.0"
+solana-rent = "3.0"
+solana-signer = "3.0"
+solana-system-interface = { version = "2.0", features = ["bincode"] }
+solana-sysvar = "3.0"
+spl-associated-token-account-interface = { version = "2.0.0", features = ["borsh"] }
```

Comment:
nit: does this actually need the borsh feature? That should only be needed for manual (de)serialization of instructions

Reply:
Good note, dropped in 53bdee7